internal/clients/flag/flag.go

```diff
@@ -0,0 +1,19 @@
+package flag
+
+import "github.com/spf13/pflag"
+
+// Values contains all values parsed from command line flags
+type Values struct {
+	Debug bool
+}
+
+// Parse parses flags from command line
+func Parse() *Values {
```

Comment:
Created a new client for parsing the flags, instead of writing a func in the main file just to keep main from growing very large. Let me know if you feel differently.